src/authfile-update

```
@@ -0,0 +1,3 @@
+#!/bin/sh
+curl https://topology.opensciencegrid.org/stashcache/authfile > /run/stashcache-cache-server-auth/Authfile.tmp
```

This seems pretty generic.  How are we going to add other VO's to this?  I figured the query would be something like:

    https://topology.opensciencegrid.org/stashcache/authfile?cache=`hostname`

---

Good point!  Addressed in such a way that the sysadmin can override the parameter (in case if the UTS name doesn't match the registration).

---

Does topology actually support that query?

---

Yup!

Now, it doesn't currently do anything with the information - just silently ignores it and gives the same output to each cache.